src/main/kotlin/app/revanced/patches/music/misc/androidauto/patch/AndroidAutoCertPatch.kt

```diff
@@ -0,0 +1,42 @@
+package app.revanced.patches.music.misc.androidauto.patch
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.addInstruction
+import app.revanced.patcher.extensions.replaceInstruction
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.music.misc.androidauto.annotations.MusicAndroidAutoCompatibility
+import app.revanced.patches.music.misc.androidauto.fingerprints.SHACertificateCheckFingerprint
+
+@Patch
+@Name("androi-dauto-cert-patch")
```

Comment:
Please, fix typo-error.

Reply:
Done when renaming according to @oSumAtrIX  suggested